packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -99,8 +98,8 @@ class RevealedOffset {
 
```

Comment:
remove the protected/visibleForTesting on defaultCacheExtent, and add the following before its "see also:
```dart
  /// This default assumes [CacheExtentStyle.pixels].
￼  ///
￼```